addons/account/models/account_account.py

```diff
@@ -901,7 +901,12 @@ def _adapt_accounts_for_account_groups(self, account_ids=None, company=None):
             company_ids = account_ids.company_id.root_id.ids
             account_ids = account_ids.ids
         else:
-            company_ids = self.company_id.ids
+            company_ids = []
+            for company in self.company_id:
+                company_ids.append(company.id)
+                if company.all_child_ids:
+                    company_ids.extend(company.all_child_ids.ids)
```

Comment:
```suggestion
            company_ids = self.company_id._accessible_branches()
```
Couldn't we simply do this?

Reply:
works perfectly with that :smile: